src/vmware/azext_vmware/custom.py

```diff
@@ -6,6 +6,20 @@
 from knack.util import CLIError
 from azext_vmware.vendored_sdks.avs_client import AVSClient
 
+LEGAL_TERMS = '''

```

Comment:
This is a long text, showing it in terminal isn't the best user experience,  today in LiftR Rps, we will show link of terms for customer reference, waht's your thinking? cc @fengzhou-msft for awareness

Reply:
@yungezz Do you have a link to that example which shows a link of terms? I'll ask the PM & CELA if a link will work.